test/fluture.test.js

```diff
@@ -6,11 +6,11 @@ const Readable = require('stream').Readable;
 const jsc = require('jsverify');
 const S = require('sanctuary');
 const FL = {
-  map: 'map',
-  bimap: 'bimap',
-  chain: 'chain',
-  ap: 'ap',
-  of: 'of'
+  map: 'fantasy-land/map',
+  bimap: 'fantasy-land/bimap',
+  chain: 'fantasy-land/chain',
+  ap: 'fantasy-land/ap',
+  of: 'fantasy-land/of'
 };
```

Comment:
> Just curious why not actually add a hard dependency?

I don't see compelling reasons to do so: https://github.com/sanctuary-js/sanctuary-type-classes/pull/2#issuecomment-247303007.